src/draggable.js

```diff
@@ -358,6 +358,7 @@
       // trigger dragend event
       if (me.dragEvent.started) {
         options.onDragEnd(element, dragEvent.x, dragEvent.y, e);
+        me.dragEvent.started = false;

```

Comment:
i think the real issue is on line 375, which breaks the referential equality when re-assigning `me.dragEvent`.